### PR TITLE
Add theme :: Seasonal design - Luna New Year

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

Add theme :: Seasonal design - Luna New Year

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**Release notes:**
- feat: add theme :: Seasonal design - Luna New Year

스크린샷 2023-01-18 오후 6 17 08
스크린샷 2023-01-18 오후 6 17 27

### CODE REVIEW

**Review by bobo-k2 — APPROVED** [review]
LGTM

File: src/components/common/themes/DividerNewYear.vue [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    #d9802e 69%,
+    #793b02 104.05%
+  );
+}
+.year {
+  font-style: normal;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 19px;
+  background: linear-gradient(180deg, #fdc605 0%, #793b02 100%);
```
Comment (Kahonnohak) [inline src/components/common/themes/DividerNewYear.vue:40]: Please change to 225deg, #d9802e, #793b02

File: src/components/common/themes/DividerNewYear.vue [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+  text-fill-color: transparent;
+  text-shadow: 0px 1px 1px rgba(0, 0, 0, 0.5);
```
Comment (Kahonnohak) [inline src/components/common/themes/DividerNewYear.vue:45]: Please remove the text shadow

**Review by Kahonnohak — CHANGES_REQUESTED** [review]

**Comment by Kahonnohak** [comment]
Hi Thank you @sirius651 

I learned how I can request changes from the code...!

Apart from the colour and the fireworks, Gold line is on top of the wallet modal on Mobile. Please take a look.

スクリーンショット 2023-01-18 14 13 26

Thanks

**Comment by sirius651** [comment]
> Thanks

@Kahonnohak Fixed above issues and adjust more about fireworks like making more smaller and set probability goes lower and increased more particles. please check again

**Comment by sirius651** [comment]
@Kahonnohak could you check again? I updated animation more by adjusting flag. but because of the size of canvas, it seems different :(

**Comment by Kahonnohak** [comment]
@sirius651 Thanks I have looked but.. Shall we remove it? we would not have time to improve.

Another small thing I noticed now - when loading the logo appears late. If you can fix that would be good otherwise just remove the fireworks please.

**Comment by sirius651** [comment]
> Another small thing I noticed now - when loading the logo appears late. If you can fix that would be good otherwise just remove the fireworks please.

Fixed it

**Comment by Kahonnohak** [comment]
Sorry...! If you have time, can you look into this? It is not good... (logo comes later than anything)
https://gyazo.com/89df545fc2dbb22e7c9d2dd3b2ba8414

**Comment by sirius651** [comment]
> Sorry...! If you have time, can you look into this? It is not good... (logo comes later than anything)
> https://gyazo.com/89df545fc2dbb22e7c9d2dd3b2ba8414

When I look at this, this is a problem currently in the portal, which seems slow because it needs to print the appropriate logo like astar, shiden according to networkInfo. Perhaps to solve this problem, the architecture needs to be fixed, so it would be better to work on a new PR.

**Comment by Kahonnohak** [comment]
You are right okay, but so sorry I found other things - the logo size is small and the location is not good (not aligned )

Could you quickly change those below please.

- Padding above the logo to 24px
- The logo size to 112%


スクリーンショット 2023-01-19 13 30 44

スクリーンショット 2023-01-19 13 31 14

**Comment by sirius651** [comment]
> You are right okay, but so sorry I found other things - the logo size is small and the location is not good (not aligned )
> 
> Could you quickly change those below please.
> 
> * Padding above the logo to 24px
> * The logo size to 112%
> 
it seems fixed now, can you check again?

**Comment by Kahonnohak** [comment]
@sirius651 and rabbit on the top of the gold line - sorry keep adding in last minute. I did not know you have day off tomorrow.

**Comment by Kahonnohak** [comment]
Yes thank you checked, and the location is a bit different from the current but if we can go back to the right place after the celebration period that is okay. 

- Shiden also need to change the logo size
- The previous post, the gold line should be behind the rabbit.

Hope you can make it...

thanks

**Comment by sirius651** [comment]
> thanks

Sorry for hurry, I believe it's fixed above?

**Comment by Kahonnohak** [comment]
Thank you!!!!! Shiden looks good but Astar rabbits is under the golden line... Is it my cache?

**Comment by Kahonnohak** [comment]
It was cache! All good thanks

**Comment by sirius651** [comment]
스크린샷 2023-01-19 오후 11 36 57

I think so, he's on the gold line :)

**Review by Kahonnohak — APPROVED** [review]
Thanks!

**Pull request merged**